docs/conditional.rst

```diff
@@ -142,7 +122,7 @@ You can check for conditional operation failures by inspecting the cause of the
 .. code-block:: python
 
     try:
-        thread_item.save(forum_name__null=False)
+        thread_item.save(Thread.forum_name.exists())
     except PutError as e:
```

Comment:
The problem here, by the way, is that both `PutError`, `DeleteError`, and `UpdateError` can have the `ConditionalCheckFailedException` error code. So I'd have to make three subclasses and users would have to choose the correct one.